client_webhooks.go

```diff
@@ -0,0 +1,58 @@
+package monta
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/url"
+)
+
+// GetWebhookConfigResponse is the response output from the [Client.GetWebhookConfig] method.
+type GetWebhookConfigResponse struct {
+	// A HTTPS URL to send the webhook payload to when an event occurs.
+	WebhookURL string `json:"webhookUrl"`
+	// A cryptoghrapic secret used to sign the webhook payload.
+	WebhookSecret string `json:"webhookSecret"`
+	// A list of event type to subscribe to. Use of ["*"] means subscribe to all.
+	EventTypes []*string `json:"eventTypes"`
+}
+
+// GetWebhookConfig to get your webhook config.
+func (c *clientImpl) GetWebhookConfig(ctx context.Context) (*GetWebhookConfigResponse, error) {
+	path := "/v1/webhooks/config"
+	query := url.Values{}
+	return doGet[GetWebhookConfigResponse](ctx, c, path, query)
+}
+
+// UpdateWebhookConfigRequest is the request input to the [Client.UpdateWebhookConfig] method.
+type UpdateWebhookConfigRequest struct {
+	// A HTTPS URL to send the webhook payload to when an event occurs.
+	WebhookURL string `json:"webhookUrl"`
+	// A cryptoghrapic secret used to sign the webhook payload.
+	WebhookSecret string `json:"webhookSecret"`
+	// A list of event type to subscribe to. Use ["*"] to subscribe to all.
+	EventTypes []*string `json:"eventTypes"`
+}
+
+// UpdateWebhookConfigResponse is the response output from the [Client.UpdateWebhookConfig] method.
+type UpdateWebhookConfigResponse struct {
```

Comment:
Could it make sense to reuse the same WebhookConfigResponse for both - Get and Put?

Reply:
I'll revisit this suggestion and make sure it's not an issue when all of the endpoints are done.